src/kodeagent/kutils.py

```
@@ -137,35 +158,44 @@ def make_user_message(
+                                'application/json', 'application/xml'
+                            ]:
+                                try:
+                                    with open(item, 'r', encoding='utf-8') as f:
+                                        file_content = f.read()
+                                    content.append(
+                                        {
+                                            'type': 'text',
+                                            'text': f'File {item} content:\n{file_content}'
+                                        }
+                                    )
+                                except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)